src/ids-alert/ids-alert.scss

```
@@ -0,0 +1,47 @@
+@import '../../node_modules/ids-identity/dist/theme-uplift/tokens/web/theme-uplift-mixins.scss';
+@import '../../node_modules/ids-identity/dist/theme-uplift/tokens/web/theme-uplift.scss';
+@import '../ids-base/ids-base';
+
+svg {
```

```suggestion
svg {
```

Probably better to do this on a class you add like ids-alert?

---

For some reason, it doesn't change the stylings unless I import it in the `ids-icon.scss`.

---

I can have a look maybe. what are you trying to change with it?

---

@tmcconechy sure, you can take a look.

---

@tmcconechy i think i found the solution for this. Will push some updates later.